### ISSUE TITLE
Jiu Jitsu 

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#879]
This evening 20:30 - 22:00 @LuchoTurtle and I went to our first Jiu Jitsu class/lesson at our local gym. 👘 
It was **`90 mins`** and I was totally _destroyed_. 🤯 
Luis is a _lot_ fitter than I am so he was fine. 💪 

Bought a Kimono (Gi) and will `try` and go 2-3 times a week for as long as I can. 3️⃣ 🤞 
It's an absolutely _brutal_ workout. I think I sprained my wrist. It's painful to type this. 🙄 
I'm a huge fan of **`functional`** / **`practical` fitness**. i.e: not just running on a treadmill ... 🏃 
We learned a decent amount about the holds, grips and movements in our first lesson. 📝 
It's clear that a _lot_ of practice will be required to **`master`** this skill. ⏳ 
A _lot_ of **`static` strength**, **stamina** and **speed** are required ... 💭 

Apparently it takes a _very_ long time [10 years] to achieve Black Belt in BJJ ... ⌛ 
I'm just going to learn as much as I can, enjoy it and get fit. 🚀

**Comment by LuchoTurtle** [comment]
[You came close to breaking my arm a few times](https://www.youtube.com/watch?v=hOZnP4dZYK0), awesome job 😂

**Comment by nelsonic** [comment]
Yeah, the arm lock could land one of us in the hospital quite easily. 🏥 
My arm is bruised too. 🙄 
Going to try and get some sleep now and see how broken I feel in the morning ... 🛌 🤞

**Comment by nelsonic** [comment]
This evening in the BJJ class when doing an overthrow, the chap I was training with - who was a little heavier than me - fell with his shoulder (and thus the full weight of his body) on my chest directly on my heart. 💔

I felt more force on my rib cage and upper chest than I’ve ever felt before in my life. Not even when the 300Kg fence/gate fell on me when I was 6 … I couldn’t breath for several minutes and my chest felt like there was a dagger in it.  🗡️

30 mins have passed and I’m sitting on the sofa at home typing this and my chest still feels like there’s a knife in it. 🔪 
I can barely breathe and my whole upper body is shaky. My heart is palpating (beating faster than resting) and breathing is mega painful.


What should I do? Should I head to the A&E to get a chest X-Ray? 🩻 
Or should I wait it out and hope that it improves?  ⏳🤷‍♂️

**Comment by iteles** [comment]
Verdict from the doctor: take an anti-inflammatory (like ibuprofen) to try to help with the pain. If as the pain improves a little you're still having trouble breathing, definitely go in for an x-ray.

**Comment by nelsonic** [comment]
I cannot breathe more than about 40%. If I breathe too deeply it’s mega painful. My heart rate is 75bpm (thanks Apple Watch) and it’s very painful to move my upper body. 

@iteles gave me a Brufen 600mg 10mins ago.
Hasn’t had any discernible effect. ⏳

**Comment by nelsonic** [comment]
@LuchoTurtle front right:
![41627C04-2D6F-418E-8194-9A7E4A5C0A25](https://user-images.githubusercontent.com/194400/196822205-bdabdabe-56ac-44a8-abb0-ab0e1f40c660.jpeg)

but where’s Waldo? 😜

**Comment by nelsonic** [comment]
Came to the emergencies @ hospital de Braga, got triaged as “yellow”:

![A2155051-8F16-4285-80FE-5DB956C6CC86](https://user-images.githubusercontent.com/194400/196823125-9f6d2059-c7e0-47cb-8bd7-660d5e169fba.jpeg)

waiting patiently while watching nature documentaries and taking notes:

![A3D8D557-A023-432A-9B6A-08BB15BB144E](https://user-images.githubusercontent.com/194400/196823619-c4954c3d-30d7-4c7e-b8b2-3b5833edefbb.jpeg)

**Comment by nelsonic** [comment]
Had 3 x-rays 🩻 
Waiting for results …
![1C0AFF7F-3692-4957-9E9E-19ED66E84D78](https://user-images.githubusercontent.com/194400/196826038-31c177fe-3b43-484c-8090-0d4020d7d98e.jpeg)

**Comment by adamjackir** [comment]
Jiu-Jitsu is all about skill, strategy, and persistence! Whether you're just starting or a seasoned grappler, there's always something new to learn. Get a Grip Jiu Jitsu is the perfect place to train, challenge yourself, and grow. Keep rolling and keep improving! Discover More: [Getagripbjj](https://getagripbjj.com/)

**Comment by iteles** [comment]
This bubbling up to the top reminded me that I have had a good few conversations lately with parents at our kids' preschool about how jiu jitsu is the best martial art for kids because it's about leveraging other people's momentum and force (sounds like aikido) rather than it being about your own strength (which obviously you lack as a kid).
However, I've never done jiu jitsue and this kind of reminds me of chinese whispers - someone said something once and then all these parents who have no clue at all about _any_ martial arts repeated it. So I was wondering what your experience of this was @nelsonic and @LuchoTurtle ?

**Comment by nelsonic** [comment]
Judo is far more appropriate to children than Jiu Jitsu at least in the first few years as there are _far_ fewer injuries.
Unless the parents discussing the topic have practiced _multiple_ martial arts and have actual _experience_ it's pointless.

Judo, Karate, Fencing, Taekwondo, BJJ, MMA & Boxing would be the order I would introduce them to _my_ child. 
I did Judo for 3 years, Karate for 6, Fencing for 2 and BJJ on-and-off as an adult.
When there are kids with vastly different strength, height and weight in the class injury is frequent in BJJ.
